Sming/Arch/Host/readme.md

```diff
@@ -90,14 +90,25 @@ At present the emulator just writes output to the console. Inputs all return 0.
 
 Support is provided via TAP network interface (a virtual network layer operating at the ethernet frame level). A TAP interface must be created first, and requires root priviledge:
 
-```
-sudo ip tuntap add dev tap0 mode tap user `whoami`
-sudo ip a a dev tap0 192.168.13.1/24
-sudo ifconfig tap0 up
-```
+	sudo ip tuntap add dev tap0 mode tap user `whoami`
+	sudo ip a a dev tap0 192.168.13.1/24
+	sudo ifconfig tap0 up
 
 This creates the `tap0` interface, and is persistent across OS reboots. The emulator will automatically select the first `tap` interface found. To override this, use the `--ifname` option. An IP address will be assigned, but can be changed using the `--ipaddr` option.

```

Comment:
> and is persistent across OS reboots

Hm. I might be wrong but IMHO that is not correct. To persist it the developers should add it to their rc.local or whatever init scripts that are executed after the network is setup.

Reply:
No, I think you're right. I'll remove it.